website/next.config.mjs

```diff
@@ -0,0 +1,20 @@
+import nextra from 'nextra'
+ 
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
```

Comment:
Comparing to here: https://github.com/Health-Informatics-UoN/rcc-monitor/blob/main/docs/next.config.js

`basePath` / `assetPrefix` might need to be set for routing/images to work properly

Reply:
I did that before, but reverted it when it didn't make a difference locally... sigh...